Specify that the contents of package.json are required (#301)
vrchat-community/creator-companion: Docs/docs/guides/create-listing.md

  * {package id}
    * versions
    * {version string in SemVer}

>       * {contents of `package.json`, which is a valid VPM manifest}

Reviewer: ```suggestion
      * {valid VPM manifest (`package.json`)}
```

Author: The reason why I changed this is that my friend misread many properties of vpm manifest are not required on the repository json. 
So, I want to explicitly show full package.json (vpm manifest) is required.
They thought properties used on installing packages (e.g. vpmDependencies, legacyFolders) are not required in repository because they are in package.json in the zip file. 
They also thought vpm manifest without optional properties is also a valid manifest and I think so.

Reviewer: Thank you for the feedback!
How's this? It's more explicit now.
```suggestion
      * {valid VPM manifest} _(the full contents of `package.json)_
```

Author: It looks good. Thank you
